recipes/orc/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "1.5.3" %}
+
+package:
+  name: orc
+  version: {{ version }}
+
+source:
+  url: http://mirrors.ibiblio.org/apache/orc/orc-{{ version }}/orc-{{ version }}.tar.gz
+  sha256: 96da3cccd2b396dc7813949d3c459a5d6995778928d344f4553cf890f171521c
+
+build:
+  number: 0
+  skip: True    # [linux32 or win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - ninja       # [unix]
+  host:
+    # putting cmake in host allows it to find the cmake modules better
+    - cmake
```

Comment:
Please, change ``cmake`` to ``build`` section and also add these following flags to your ``cmake``
```
    -DCMAKE_PREFIX_PATH=$PREFIX \
    -DCMAKE_INSTALL_PREFIX=$PREFIX \
```

Reply:
Done